rust-runtime/inlineable/src/idempotency_token.rs

```diff
@@ -94,3 +94,11 @@ impl Clone for IdempotencyTokenProvider {
         }
     }
 }
+
+impl std::fmt::Debug for IdempotencyTokenProvider {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("IdempotencyTokenProvider")
+            .field("inner", &"** redacted **")
```

Comment:
What's the reason for redacting this? I don't think its sensitive at all.

Reply:
I don't think this is sensitive at all. I would just use the auto-derived `Debug` implementation.

Reply:
Thanks guys for confirming. Switched to `#[derive(Debug)]` in 96db96f.